.vscode/settings.json

```diff
@@ -21,10 +21,9 @@
     "Scalebar",
     "Struct",
     "subclassing",
-    "toolkitcpp",
-    "toolkitqml",
+    "toolkit",
```

Comment:
I'm not sure how this JSON is used. Maybe it's better to keep "toolkitcpp"? @jared-2016

Reply:
quick google, looks like project specific configuration that adds these items to the spellcheck. Im guessing so these items don't get the red squiggly line underneath them. Maybe do a quick search of the toolkit repo with `toolkitcpp` and see if there are a lot of hits and decide from there?

Reply:
After a closer look I see that `toolkitcpp` is still in use, we should probably leave that one in until we remove it's usage from the repo.